src/components/WebexAudioSettings/WebexAudioSettings.jsx

```diff
@@ -18,14 +18,21 @@ import WebexNoMedia from '../WebexNoMedia/WebexNoMedia';
 export default function WebexAudioSettings({className, meetingID, style}) {
   const cssClasses = webexComponentClasses('audio-settings', className);
   const [, display] = useMeetingControl('switch-microphone', meetingID);
+  // The browser api setSinkId() does not work properly on Firefox and Safari browsers.
+  // https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/setSinkId
+  const isFirefox = navigator.userAgent.indexOf('Firefox') !== -1; // detect Firefox browser
+  const isSafari = navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1; // detect Safari browser
```

Comment:
I think you should move these into the [helpers module](https://github.com/webex/components/blob/master/src/components/helpers.js), they could be useful for other components in the future.
You would make them into functions instead, e.g. `isFirefox()` `isSafari()` and they would return a boolean